README.md

```diff
@@ -42,16 +72,34 @@ To install the plugin, run:
   - `status <CR>` : Print the phase and conditions of CRs of a specific type, such as `cephobjectstore`, `cephfilesystem`, etc
   - `purge-osd <osd-id> [--force]` : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
 
-- `debug` : [Debug a deployment](#debug-mode)  by scaling it down and creating a debug copy. This is supported for mons and OSDs only
-  - `start  <deployment-name> `
+- `debug` : [Debug a deployment](./docs/debug.md)  by scaling it down and creating a debug copy. This is supported for mons and OSDs only
+  - `start  <deployment-name>`
     `[--alternate-image <alternate-image>]` : Start debugging a deployment with an optional alternative ceph container image
   - `stop  <deployment-name>` : Stop debugging a deployment
 
 - `help` : Output help text
 
-## Examples
+## Documentation
+
+Visit docs below for complete details about each command and their flags uses.
+
+1. [Running ceph commands](docs/ceph.md)
+1. [Running rbd commands](docs/rbd.md)
+1. [Getting mon endpoints](docs/mons.md)
+1. [Get cluster health status](docs/health.md)
+1. [Update configmap rook-ceph-operator-config](docs/operator.md#set)
+1. [Restart operator pod](docs/operator.md#restart)
+1. [Get rook version](docs/rook.md#version)
+1. [Get all CR status](docs/rook.md#status-all)
+1. [Get cephCluster CR status](docs/rook.md#status)
+1. [Get specific CR status](docs/rook.md#status-cr-name)
+1. [To purge OSD](docs/rook.md#operator.md)
+1. [To start Debug OSDs/Mons](docs/debug.md#start-debug-mode)
+1. [To stop Debug OSDs/Mons](docs/debug.md#stop-debug-mode)
 
-### Run a Ceph Command
+### Few Examples
```

Comment:
Let's raise the level of header for examples
```suggestion
## Examples
```

Reply:
ack